internal/command/migrate_to_v2/pg.go

```diff
@@ -92,20 +92,19 @@ func (m *v2PlatformMigrator) migratePgVolumes(ctx context.Context) error {
 		fmt.Fprintf(m.io.Out, "Creatings %d new volume(s) in '%s'\n", len(vols), region)
 		for _, vol := range vols {
 			// TODO: make use of https://github.com/superfly/nomad-firecracker/pull/1013
-			input := api.CreateVolumeInput{
-				AppID:     app.ID,
+			input := api.CreateVolumeRequest{
 				Name:      fmt.Sprintf("%s_machines", vol.Name),
 				Region:    region,
-				SizeGb:    vol.SizeGb,
-				Encrypted: vol.Encrypted,
+				SizeGb:    &vol.SizeGb,
+				Encrypted: api.Pointer(vol.Encrypted),
```

Comment:
Can we address this comment here: 
> // TODO: make use of https://github.com/superfly/nomad-firecracker/pull/1013

Reply:
or ignore it. I introduced a bug when removing `_machines` suffix and can fix it in another PR.